golang/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Golang versions
 
-All golang services (with some exceptions) at Clever should be on go-1.5.
+All golang services (with some exceptions, like Heka) at Clever should be on go-1.5.
```

Comment:
Reasonable to get a list of current exceptions, or what informal criteria are used for making exceptions?